scripts/zones/Attohwa_Chasm/mobs/Tiamat.lua

```diff
@@ -59,32 +59,74 @@ local spawnPoints =
     { x = -549.802, y = -8.944,  z = -24.848 },
 }
 
-local function setupFlightMode(mob, battleTime, mobHP)
+local function enterFlight(mob)
     mob:setAnimationSub(1) -- Change to flight.
     mob:addStatusEffectEx(xi.effect.ALL_MISS, 0, 1, 0, 0)
     mob:setMobSkillAttack(730)
-    mob:setLocalVar('changeTime', battleTime)
-    mob:setLocalVar('changeHP', mobHP / 1000)
+    mob:setLocalVar('flightTime', os.time() + 120)
+    mob:setLocalVar('changeHP', mob:getHP() - 10000)
 end
 
 entity.onMobInitialize = function(mob)
     mob:setCarefulPathing(true) -- Used for drawin
 
     xi.mob.updateNMSpawnPoint(mob, spawnPoints)
-    mob:setRespawnTime(math.random(86400, 259200)) -- When server restarts, reset timer
+    mob:setRespawnTime(math.random(144, 240) * 1800) -- 3 to 5 days in 30 minute windows
 end
 
 entity.onMobSpawn = function(mob)
-    -- Reset animation so it starts grounded.
+    -- Ensure Tiamat spawns with correct ground status
     mob:setMobSkillAttack(0)
     mob:setAnimationSub(0)
+    mob:delStatusEffect(xi.effect.ALL_MISS)
     mob:setMobMod(xi.mobMod.NO_MOVE, 0)
+
+    mob:setMod(xi.mod.ACC, 444)
+    mob:setMod(xi.mod.ATT, 388)
+    mob:setMod(xi.mod.COUNTER, 10)
+    mob:setMod(xi.mod.CURSERES, 100)
```

Comment:
Do we have a cap that shows the message `Resist! Tiamat resists the effect...` ?
Or is it we need a curse immunity?

Not the same thing

Reply:
Getting some help with more captures about this behavior. Frank was wondering if all NMs are curse immune, but doesn't seem to be that way right now.

Reply:
Well, until then, use xi.mod.CURSE_MEVA and set it to 1000 or something.

Reply:
probably should make a curse/doom immunity, we'll need it for Cruel Joke (doom)

Reply:
Made an issue to track this better #6947

Reply:
I added CURSE_MEVA in place of the CURSERES. Siknoz was able to get some testing on Vrtra at least and never saw curse proc once. https://1drv.ms/u/c/87e665e10555c452/EZSsQPm24ZxBhFxIyL-qU04BD0_9cGbrIzzZjy9A_ZQSJg?e=U8lc3H

Of course SE has curse immune on some NMs and not on others. I am assuming all three of these wyrms are immune to it though, but I put a comment on saying it needs to be verified for Tiamat and Jorm still.

Reply:
yeah that works for me, we just need a better backend to support it which is a bit out of scope for this PR